transforms/universal/ededup/python/README.md

```diff
@@ -12,7 +15,11 @@ This is a python version of ededup
 The set of dictionary keys holding [EdedupPythonTransform](src/ededup_transform_python.py) 
 configuration for values are as follows:
 
-* _ededup_doc_column_ - specifies the name of the column containing a document
+* _doc_column_ - specifies the name of the column containing a document
+* _doc_id_column_ - specifies the name of the column containing a document id
+* _use_snapshot_ - specifies that ededup execution starts from a set of already seen hashes. This can be used
```

Comment:
can we not have multiple terms for the same thing.  How about incremental=True instead of use_snapshot=True.

Reply:
Incremental is one case of using transforms

Reply:
i don't believe that speaks to my point.  having multiple terms is just plain obfuscation.  can we pick one please?

Reply:
We are picking one - snapshotting. It is used for incremental, among other things